src/components/preview/components/page-preview-text-view.tsx

```diff
@@ -104,6 +136,24 @@ class PagePreviewTextView extends React.Component<Props, State> {
       }
 
       switch (property.name) {
+        case "width": {
+          const px = AndroidUtils.stringToPx(displayMetrics, property.value, scale);
+          if (px) {
+            result.width = px;
+          } else {
+            console.log("Button: unknown width", property.value);
+          }
+          break;
+        }
+        case "height": {
+          const px = AndroidUtils.stringToPx(displayMetrics, property.value, scale);
+          if (px) {
+            result.height = px;
+          } else {
+            console.log("Button: unknown height", property.value);
+          }
+          break;
+        }
```

Comment:
switch case formatting goes like this:

```suggestion
        case "width":
          const px = AndroidUtils.stringToPx(displayMetrics, property.value, scale);
          if (px) {
            result.width = px;
          } else {
            console.log("Button: unknown width", property.value);
          }
        break;
        case "height":
          const px = AndroidUtils.stringToPx(displayMetrics, property.value, scale);
          if (px) {
            result.height = px;
          } else {
            console.log("Button: unknown height", property.value);
          }
        break;
```